pkg/scheduler/objects/queue_test.go

```diff
@@ -1334,6 +1334,17 @@ func TestGetPartitionQueueDAOInfo(t *testing.T) {
 	root.guaranteedResource = getResource(t)
 	assert.DeepEqual(t, root.GetMaxResource().DAOMap(), root.GetMaxResource().DAOMap())
 	assert.DeepEqual(t, root.GetGuaranteedResource().DAOMap(), root.GetGuaranteedResource().DAOMap())
+
+	// test allocatingAcceptedApps
+	root.allocatingAcceptedApps = getAllocatingAcceptedApps()
+	assert.Assert(t, reflect.DeepEqual(len(root.allocatingAcceptedApps), len(root.GetPartitionQueueDAOInfo().AllocatingAcceptedApps)))
```

Comment:
Good point. 
I updated to check root.allocatingAcceptedApps and root.GetPartitionQueueDAOInfo().AllocatingAcceptedApps lengths separately since they might be same. 
I also checked root.GetPartitionQueueDAOInfo().AllocatingAcceptedApps element value it should be appID1 since it's true, we don't append appID2 since it's false.